Dockerfile

```diff
@@ -1,33 +1,17 @@
 # Used to host the isomorphic web app example
 
-FROM rust:latest as build
-
-WORKDIR /usr/src
-
-COPY . .
+FROM yasuyuky/rust-nightly-musl:latest as build
 
 # Install wasm32-unknown-unknown-target
 RUN rustup default nightly
 RUN rustup target add wasm32-unknown-unknown
 RUN rustup target add x86_64-unknown-linux-musl
 
-# Build example isomorphic server binary
-RUN cargo build -p isomorphic-server --release --target x86_64-unknown-linux-musl
-
 # Node.js & npm
 RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
-RUN apt-get install -y nodejs
-
-# Build tools for making npm install work
-RUN apt-get install -y build-essential
-RUN apt-get install -y libssl-dev
-RUN apt-get install -y pkg-config
-
-# Ge npm package dependencies
-RUN npm install
-
-# Compile to WASM
-RUN cargo build -p isomorphic-client --release --target wasm32-unknown-unknown
+RUN apt-get install -y nodejs \
+  # Build tools for making npm install work 
```

Comment:
FWIW I copied this from an old Dockerfile that I had and have no idea if there are still necessary.. but probably not worth looking into since it works. Just passing that info along

Reply:
Seems the whole lot's not needed.